tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/PacketTracer.csproj

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
```

Comment:
How long does this run? Should it possibly be priority 1, i.e.:
```
    <CLRTestPriority>1</CLRTestPriority>
```
Also, I took a brief look at the test, and I think it should be wrapped in a try/catch so that we don't consider it passing if it throws an exception.

Reply:
Could you file an issue so that this doesn't get forgotten?
I'm not concerned about catching PNSE - the point is to catch any unexpected exception and report failure.

Reply:
@CarolEidt Thank you for the suggestions, logged at https://github.com/dotnet/coreclr/issues/20557. I will submit a PR by the end of this week.

Reply:
Thanks!